### DISCUSSION TITLE
Extracting firmware signature failure

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#264] [General]
I'm moving an rnode to a new system and figured I'd extract the firmware for no real reason other than to try it out. It failed due to the signature not being valid.*

*(As noted in a previous thread, it's because it's a new machine which lacks the signing key)

I see no option to force the extraction. While I understand the reasoning behind this safety check, I can think of no use case where you'd need to extract firmware to a machine with the signing key. If I recall there's a way to access the stored data in other ways for turning a seed into a forest (please forgive me if I can't recall the exact wording) but extracting a known-good firmware to another machine through rnodeconf seems like a good idea to me.

Is this functioning as intended?

**Comment by markqvist** [comment]
Hmm, that should definitely be possible, exactly for the reasons mentioned. I am going to transfer this as a high-priority issue to the tracker.
